### PR TITLE
Elevio Integration

### PR DESCRIPTION
**Pull request by ezequielc**
## Description
Replaces Zendesk widget with Elev.io widget.

I will work on unit tests once Brian confirms they are going to proceed with that service.

## Motivation and Context
Request from Support team.

## How Has This Been Tested?
Manually tested locally and deployed on apps-beta for Brian's validation.

## Release Plan:
- As the Submitter, upon requesting review of this pull request, I confirm that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed. 
- As the Reviewer, upon approving the changes in this PR, I confirm I have reviewed and I agree that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed

#### Release Checklist Items Skipped?
If any Release Checklist items were intentionally skipped, please provide which ones and the reasons why

### CODE REVIEW

**Comment by ezequielc** [comment]
@alex-deaconu I will open a new PR for unit tests and get ready to release this today. Thanks!

**Comment by alex-deaconu** [comment]
@ezequielc why is the test coverage so low I don't see tests being disabled. Thanks.

File: web/scripts/common-header/services/svc-help-widget.js [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+            $window.document.body.appendChild(scriptElem);
+            $window._elev.on('load', function(_elev) {
+              _elev.setSettings({
+                hideLauncher: true
+              });
+            });
+            loaded = true;
+          }
+          return $q.when();
```
Comment (alex-deaconu) [inline web/scripts/common-header/services/svc-help-widget.js:27]: @ezequielc why are we returning `$q.when()`? Are we trying to make this return asynchronous? Why not just return `$q.resolve()` as in other places. Thanks.

File: web/scripts/common-header/services/svc-help-widget.js [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+      function ($q, $window, userState, HELP_WIDGET_SCRIPT) {
+        var loaded = false;
+
+        function ensureScript() {
+          if (!loaded) {
+            var scriptElem = $window.document.createElement('script');
+            scriptElem.innerText = HELP_WIDGET_SCRIPT;
+
+            $window.document.body.appendChild(scriptElem);
+            $window._elev.on('load', function(_elev) {
```
Comment (alex-deaconu) [inline web/scripts/common-header/services/svc-help-widget.js:20]: @ezequielc to confirm, there's no `identify` call for this widget? Thanks.

File: web/scripts/common-header/services/svc-help-widget.js [hunk 3]
```diff
@@ -0,0 +1,96 @@
+/* jshint maxlen: false */
+
+(function (angular) {
+  'use strict';
+
+  angular.module('risevision.common.support', [])
+    .value('HELP_WIDGET_SCRIPT',
+      '!function(e,l,v,i,o,n){e[i]||(e[i]={}),e[i].account_id=n;var g,h;g=l.createElement(v),g.type="text/javascript",g.async=1,g.src=o+n,h=l.getElementsByTagName(v)[0],h.parentNode.insertBefore(g,h);e[i].q=[];e[i].on=function(z,y){e[i].q.push([z,y])}}(window,document,"script","_elev","https://cdn.elev.io/sdk/bootloader/v4/elevio-bootloader.js?cid=","5f2331387a97f");'
+    )
+    .factory('helpWidget', ['$q', '$window', 'userState', 'HELP_WIDGET_SCRIPT',
```
Comment (alex-deaconu) [inline web/scripts/common-header/services/svc-help-widget.js:10]: @ezequielc should we rename to helpWidgetFactory to indicate this is a factory? Thanks.

File: web/scripts/common-header/controllers/ctr-help-widget-button.js [hunk 4]
```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+angular.module('risevision.common.header')
+  .controller('HelpWidgetButtonCtrl', ['$scope', 'helpWidget',
+    function ($scope, helpWidget) {
+      $scope.showHelpWidget = function () {
+        helpWidget.activateWidget();
```
Comment (alex-deaconu) [inline web/scripts/common-header/controllers/ctr-help-widget-button.js:7]: @ezequielc I would name these the same or just use the factory in the partial. Do we even need a controller? Maybe we can just load the factory in the parent scope. Just some thoughts. Thanks.

Comment (ezequielc) [reply web/scripts/common-header/controllers/ctr-help-widget-button.js:7]: @alex-deaconu This was an existing code, I just renamed it from `ZendeskButtonCtrl` to `HelpWidgetButtonCtrl` and updated method names. I agree we should probably use the factory in the partial and eliminate this. I will try to do that in my next PR. Thanks

File: web/scripts/common-header/services/svc-help-widget.js (hunk 3 above)
Comment (ezequielc) [reply web/scripts/common-header/services/svc-help-widget.js:10]: @alex-deaconu Good point, I will update. Thanks

File: web/scripts/common-header/services/svc-help-widget.js (hunk 2 above)
Comment (ezequielc) [reply web/scripts/common-header/services/svc-help-widget.js:20]: @alex-deaconu That is correct. Thanks

File: web/scripts/common-header/services/svc-help-widget.js (hunk 1 above)
Comment (ezequielc) [reply web/scripts/common-header/services/svc-help-widget.js:27]: @alex-deaconu This was also existing code from Zendesk factory. I don't think we need the asynchronous return anymore. I will update it. Thanks

**Comment by ezequielc** [comment]
@alex-deaconu I will address the comments in a new PR and will double check what happened to unit tests coverage. Thanks

File: web/scripts/common-header/services/svc-help-widget.js (hunk 2 above)
Comment (alex-deaconu) [reply web/scripts/common-header/services/svc-help-widget.js:20]: Sounds good, thanks!

**Pull request merged**